tests/tensor_search/test_bulk_search.py

```diff
@@ -59,6 +60,22 @@ def test_bulk_search_w_extra_parameters__raise_exception(self):
                 "parameter-not-expected": 1,
             }]))
 
+    @mock.patch.dict('os.environ', {**os.environ, **{'MARQO_MAX_SEARCHABLE_TENSOR_ATTRIBUTES': '0'}})
```

Comment:
can we test with different cases of `MARQO_MAX_SEARCHABLE_TENSOR_ATTRIBUTES `? For example: 0, 1, 2, 10, None?

Reply:
And cases where it fails/passes?

Reply:
We can, but I don't see the point.

Reply:
it's a one-off cost (that can be minimised by copy-pasting) for stronger guarantees about how this function works indefinitely into the future